### DISCUSSION TITLE
Dose calculation at a specific voxel

### DISCUSSION THREAD

**Discussion by bcmohamed** [OpenTOPAS/OpenTOPAS#109] [Feature-Request] [answered]
Hello, how can I fix a voxel in the phantom? For example, on the z axis, the voxel of z from 1.4 cm to 1.5 cm. The dose calculation is done only in this voxel and the dose display is only for this voxel
Thanks

**Comment by LouisLechNissen** [comment, marked as answer]
Hi,

Instead of scoring in whatever (I assumed binned) geometry you are currently working with, you could make a new geometry and assign it to a parallel world. These can be used for scoring without interfering with the physics. See the documentation for how to do so :)

Best regards,
Louis

**Comment by tmasilela** [comment]
Hi,

As noted by Louis - this is probably the easiest solution if you don't want to write any code. Relevant section in the documentation [here](https://opentopas.readthedocs.io/en/latest/parameters/geometry/parallel_world.html).

An alternative solution would be to write a short python script to check specific voxels in your output file. This is more flexible and the coding requirement is very minimal.

All the best,
Thongchai
